### PR TITLE
create branchtesting

### PR DESCRIPTION
**Pull request by ademolaomosanya**
text description

### CODE REVIEW

**Review by ah-med — APPROVED** [review]
Nice work ! 
Good Job

**Review by ah-med — CHANGES_REQUESTED** [review]
You need to fix this

File: branchtesting/branch.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Branchtesting</title>
```
Comment (ah-med) [inline branchtesting/branch.html]: put space here it should be "Branch Testing" instead

**Review by ah-med — CHANGES_REQUESTED** [review]

File: branchtesting/branch.html (hunk 1 above)
Comment (ademolaomosanya) [reply branchtesting/branch.html]: done

**Review by ah-med — APPROVED** [review]

**Pull request merged**